lib/index.js

```diff
@@ -94,44 +93,16 @@ bedrock.events.on('bedrock-cli.ready', () => {
 function init(callback) {
   const config = bedrock.config.mongodb;
 
-  if(config.url) {
-    // update broken-down config values from URL
-    const parsed = mongodbUri.parse(config.url);
-    if('database' in parsed) {
-      config.name = parsed.database;
-    } else {
-      // ensure database is set
-      parsed.database = config.name;
-    }
-    if(parsed.hosts.length > 0) {
-      config.host = parsed.hosts[0].host;
-      if('port' in parsed.hosts[0]) {
-        config.port = parsed.hosts[0].port;
-      } else {
-        // default mongodb port
-        config.port = 27017;
-      }
-    }
-    if('username' in parsed) {
-      config.username = parsed.username;
-      delete parsed.username;
+  if(!config.url) {
+    config.url = 'mongodb://';
+    if(config.username) {
+      config.url += `${config.username}:${config.password}@`;
     }
-    if('password' in parsed) {
-      config.password = parsed.password;
-      delete parsed.password;
+    config.url += `${config.host}:${config.port}/${config.name}`;
+    if(config.username) {
+      // TODO: formalize support for alternate authSource
```

Comment:
Can we get this TODO removed now? If not, can it be removed and turned into an issue please?

Reply:
ok so here is what this refers to:

```js
config.mongodb.username = undefined;
config.mongodb.password = undefined;
config.mongodb.adminPrompt = true;
```

however in order to login with a username and password you now need an authSource.
Right now `authSource` is set in `config.connectOptions`. So the question is do we want to move authSource down so it is not nested in connectOptions, but instead a root level config option the same was `username` and `password` are?